test/tests/materials/interface_material/interface_value_material.i

```diff
@@ -0,0 +1,248 @@
+[Mesh]
```

Comment:
this is the input file that if used will generate a warning because the neighbor material property are not pulled correctly.